tests/unit/_base/all.js

```diff
@@ -4,6 +4,7 @@ define([
 	'./array',
 	'./connect',
 	'./declare',
+	'./declare-strict',
```

Comment:
@msssk Code targeting IE8 and previous should not use the new syntax as it may break due to JScript bugs NFE hoisting.